lib/ohai/plugins/linux/platform.rb

```diff
@@ -113,6 +129,15 @@ def os_release_file_is_cisco?
       # no way to determine platform_version in a rolling release distribution
       # kernel release will be used - ex. 3.13
       platform_version `uname -r`.strip
+    elsif (os_release_info = os_release_file_is_cisco?)
+      # Cisco platform not based on known distro above
+      if os_release_info['ID'] == 'nexus'
+        platform 'nexus'
+        platform_family 'cisco-wrlinux'
+      elsif os_release_info['ID_LIKE'] =~ /cisco-wrlinux/
+        platform_family 'cisco-wrlinux'
```

Comment:
We were asked to change from 'wrlinux' to 'cisco-wrlinux' just in case anyone else is using their own wrlinux variant that doesn't have the same assumptions as Cisco's wrlinux variants. I don't know what the risk is but we were asked to make this change.